findimports.py

```diff
@@ -51,6 +51,8 @@
                         nothing are removed.
   -D MAX_DEPTH, --depth MAX_DEPTH
                         import depth in ast tree. Default: no limit
+  --attrs ATTRIBUTES    Add custom attributes to dot graph.
+                        Specified as json string.
```

Comment:
Wouldn't it be more convenient to do `-a rank=TB` instead of `--attrs '{"rank": "TB"}'`?  Spelling out JSON on the command line is a pain.

Reply:
Hm, depends on how many attributes. If it is just one or 2 I go with multiple-argument flag, if it can be more of a small config size, I use JSON and quite happy about it. But I don't see this used often so I am ok with your suggestion as well.